newton/tests/test_examples.py

```diff
@@ -325,6 +327,76 @@ class TestRobotExamples(unittest.TestCase):
 )
 
 
+class TestRobotPolicyExamples(unittest.TestCase):
+    pass
+
+
+add_example_test(
+    TestRobotPolicyExamples,
+    name="robot.example_robot_policy",
+    devices=cuda_test_devices,
+    test_options={"num_frames": 500, "torch_required": True, "robot": "g1_29dof"},
+    test_options_cpu={"num_frames": 10},
+    use_viewer=True,
```

Comment:
Should be `test_suffix="G1_29dof"`. Also can remove the `test_options_cpu` argument.